### DISCUSSION TITLE
Doctests and inclusion in redis.io doc pages

### DISCUSSION THREAD

**Discussion by SoulPancake** [redis/go-redis#2602] [General]
@elena-kolevska Are the doctests being added as a way to include it in code examples for the documentations?
![image](https://github.com/redis/go-redis/assets/70265851/2b48021e-ed86-404c-8da1-7718a2005b41)

like above?

Asking as I'm interested in contributing to this as well as it seems like a cross-repo task.

**Comment by elena-kolevska** [comment]
Yes, that's exactly it. We're slowly going to start adding more examples for commands and tutorials. You can read more about the process in the readme file https://github.com/redis/go-redis/blob/84f7f0ff4fa8f630df05d0645a7af8720461e218/doctests/README.md

**Reply by SoulPancake** [reply]
Will this be added for all the commands?
Which all commands/tutorials will include these? Is there a way to track that?

**Reply by elena-kolevska** [reply]
Yes, it's coming very soon, probably next week! I'll keep you posted.

**Reply by SoulPancake** [reply]
Regarding the same, Can we open PRs to your branch regarding for more doctests or should we await it being merged and then eventually more PRs?

**Reply by elena-kolevska** [reply]
It would be great if we could count on you for some help with this, for sure! But the plan is not ready, I'm currently working on it. Most probably, we're gonna start with the data types tutorials first (this page: https://redis.io/docs/data-types/tutorial/).
I'll be reaching out! 
Thanks for offering help! :)

**Reply by SoulPancake** [reply]
Understood, Thanks! ^_^
